docs/api.md

```diff
@@ -40,7 +40,7 @@ Fine-grain resource evolution alone makes it difficult to eliminate fields or re
 
 Distinct API versions present more clear, consistent views of system resources and behavior than intermingled, independently evolved resources. They also provide a more straightforward mechanism for controlling access to end-of-lifed and/or experimental APIs.
 
-The [API and release versioning proposal](versioning.md) describes the current thinking on the API version evolution process.
+The [API and release versioning proposal](../docs/design/versioning.md) describes the current thinking on the API version evolution process.
```

Comment:
I though the mungedocs link checker would normalize this to design/versioning.md ?  did you run verify-gendocs?